src/app/modules/tooltip/tooltip.component.scss

```diff
@@ -36,7 +36,7 @@
 }
 
 .tooltip-message {
-  font-family: Roboto, sans-serif;
+  font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
Best to change to variables here and similar places.

Reply:
agree